.github/workflows/ci.yml

```diff
@@ -130,6 +130,12 @@ jobs:
           CAKE_TEST_AUTOQUOTE=1 vendor/bin/phpunit --testsuite=database
         fi
 
+    - name: Prefer lowest check
+      run: |
+        if ${{ matrix.prefer-lowest == 'prefer-lowest' }}; then
+          composer lowest-setup && composer lowest
```

Comment:
Why run this separate from the composer install step?

Reply:
We could also manually composer require of course, I just wanted to quickly reuse the existing developer script commands available.

Reply:
I meant this could just be the composer install step for prefer-lowest.

Reply:
Yes, as I said: I was just reusing the existing scripts here, but of course manual one works.
I am fine either way.